dev-docs/RFCs/v1.0/tentative-feature.md

```diff
@@ -0,0 +1,59 @@
+# Tentative Features
+
+In order to better support Multi* geometries, I think we'll need a concept like "tentative feature". For example, when the user adds two points of what will be a second polygon of a MultiPolygon feature. This concept would only apply to the draw* modes.
```

Comment:
I prefer having an RFC written in more of a definite proposal style, rather than as a discussion document. It is hard to approve a discussion. See the guidelines.

**Summary** - This RFC proposes adding support for "tentative features", that enables features being created or changed by editing but may be removed e.g. as the operation is cancelled.